admin_solutions/user_role_mgmt.adoc

```diff
@@ -0,0 +1,534 @@
+link:../admin_solutions/user_role_mgmt.html#view-roles-users-project[each project] or
+link:../admin_solutions/user_role_mgmt.html#view-roles-users-cluster[the entire cluster].
+
+=== Leveraging Default Groups
+
+There are special groups that are assigned to users. You can target users with these groups, but you cannot modify them. These special groups are as follows:
+
+[options="header"]
+|===
+
+|Group |Description
+
+|`system:authenticated`
+|This is assigned to all users who are identifiable to the API. Everyone who is not `system:anonymous` (the user) is in this group.
+
+|`system:authenticated:oauth`
+|This is assigned to all users who have identified using an oauth token issued by the embedded oauth server. This is not applied to service accounts (they use service account tokens), or certificate users.
+
+|`system:unauthenticated`
+|This is assigned to users who have not presented credentials. Invalid credentials are rejected with a 401 error, so this is specifically users who did not try to authenticate at all.
+|===
+
+You may find it helpful to target users with the special groups listed above. For example, you could share a pod with all users by granting `system:authenticated` access to the pod.
+
+The "default" permissions of users are defined by which roles are bound to the
+`system:authenticated` and `sytem:authenticated:oauth` groups. As mentioned
+above, you are not able to modify membership to these groups, but you can
+link:../admin_guide/manage_authorization_policy.html#managing-role-bindings[change the roles bound to these groups]. For example, to bind a role to the `system:authenticated` group for all projects in the cluster:
+
+----
+$ oadm policy add-cluster-role-to-group <role> system:authenticated
+----
+
+Currently, by default the `system:authenticated` and `sytem:authenticated:oauth`
+groups receive the following roles:
+
+[options="header"]
+|===
+
+|Role |Description
+
+|`shared-resource-viewer`
+|For the `openshift` project: allows users to see templates and pull images.
+
+|`basic-user`
+|For the the entire cluster: allows users to see their own account, check for information about requesting projects, see which projects they can view, and check their own permissions.
+
+|`self-provisioner`
+|Allows users to request projects.
+
+|`system:oauth-token-deleter`
+|Allows users to delete any oauth token for which they know the details.
+
+|`cluster-status`
+|Allows users to see which APIs are enabled, and basic API server information such as versions.
+
+|`system:webhook`
+|Allows users to hit the webhooks for a build if they have enough additional information.
+|===
+
+[[view-roles-users-project]]
+=== Viewing Roles and Users for a Project
+You can view a list of all the users that are bound to the project, and the
+roles to which they are bound. To view permissions per-project:
```

Comment:
Might be a case of bound-ness. I'd suggest:

"To view a list of all users that are bound to a project and their roles:"